docs/registry.md

```diff
@@ -18,7 +18,7 @@ Extra storage data:
 We store these two `big_map`s separately because we want to include `proposalId` for deleted keys as well.
```

Comment:
I don't think that any of this paragraph nor the info above here still holds true. 

AFAICT there aren't any `big_map`s anymore, are there?

Reply:
Oh, missed it. Changed "big_map" to "map" now.

Reply:
Come to think of it, should these be changed to `big_map`s instead? I think that there is the possibility that either of these `map`s will grow too much, making the contract unusable, is that correct?

I see that this wasn't quire introduced here, so I'm fine with leaving this for a separate issue.

However I was also referring to the rest of the doc too, where I see that much of the explanation revolves around arbitrary `k` and `v` types that are supposed to be represented by `bytes`, but that's actually not the case (AFAICT they are `string`s and in fact have been so for a long time). Am I correct? Can you please fix that?

Reply:
@sras not sure if you saw the comment just above

Reply:
I have included that in the built in registry, `k` and `v` are set as strings. Is that good enough? Or should we remove mentioning these as parameteric and use `string` everywhere?

Reply:
Yep, that work for me.

I still think that we should consider switching to use `big_map`, but I think that can be left for a separate issue/MR to avoid pushing this one even further.

Reply:
Have created this issue https://github.com/tezos-commons/baseDAO/issues/322